src/main/java/frc/robot/subsystems/Hopper.java

```diff
@@ -0,0 +1,56 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.LoadingConstants;
+
+public class Hopper extends SubsystemBase {
+
+  private final TalonSRX m_hopperMotor;
+
+  /**
+   * Creates a new Hopper.
+   */
+  public Hopper() {
+    m_hopperMotor = new TalonSRX(LoadingConstants.kLoadingMotorPort);
+  
+    motorInit(m_hopperMotor);
+    
+  }
+
+  public void motorInit(TalonSRX motor) {
+    motor.configFactoryDefault(); // configures the factory defaults in case and changes were previously made to the motor  
```

Comment:
I would move this comment to a new line to make it easier to read.
```suggestion
    // configures the factory defaults in case and changes were previously made to the motor
    motor.configFactoryDefault();  
```

Reply:
Moved the comment to be above the line of code